pawsitive/config/routes.rb

```diff
@@ -11,4 +11,6 @@
 
   # Defines the root path route ("/")
    root "home#index"
+   delete '/sessions', to: 'sessions#destroy'
+
```

Comment:
Hello @Gladdy17,it is not necessary to explicitly provide a route to delete it is already a devise helper function just specify the button_to.